betelgeuse.py

```diff
@@ -261,7 +262,7 @@ def add_test_case(args):
                     test.name,
                     test.docstring if test.docstring else '',
                     caseautomation=auto_status,
-                    casecomponent='-',
+                    casecomponent=casecomponent,
```

Comment:
Based on the previous comment, there is no need to create a single use variable for case component, it can be done here:

```python
...(
    ...,
    casecomponent=test.unexpected_tags.get('casecomponent', '-')
    ...
)
```